.github/workflows/ruleset-doc-test.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
   lint:
     name: Doctest
     # You do not appear to be able to use variables in the "uses" field.
-    uses: NRLMMD-GEOIPS/geoips_ci/.github/workflows/reusable-doc-test.yaml@main
+    uses: NRLMMD-GEOIPS/geoips_ci/.github/workflows/reusable-doc-test.yaml@build_stable_image_on_tag
```

Comment:
NOTE this will need to be backed out, and the repository ruleset updated to point to main again

Reply:
I changed this to main.

Reply:
https://github.com/NRLMMD-GEOIPS/geoips_ci/pull/15/files#r1929425962

Reply:
https://github.com/NRLMMD-GEOIPS/geoips_ci/pull/15/commits/f5a80edc95d4284fd09d0931bba6070503c2695c